typebinder/src/type_solving/solvers/ranges.rs

```diff
@@ -0,0 +1,86 @@
+use super::path::PathSolver;
+use crate::{
+    contexts::exporter::ExporterContext,
+    error::TsExportError,
+    type_solving::{
+        fn_solver::AsFnSolver,
+        result::{Solved, SolverResult},
+        type_info::TypeInfo,
+        TypeSolver, TypeSolverExt,
+    },
+    utils::inner_generic::solve_segment_generics,
+};
+use syn::Type;
+use ts_json_subset::types::{
+    ObjectType, PrimaryType, PropertyName, PropertySignature, TsType, TypeBody, TypeMember,
+};
+
+/// A solver that solves Ranges.
```

Comment:
Note to self: would be nice if all std-supported types could be listed in the docs with direct links to the Rust stdlib, like https://doc.rust-lang.org/std/ops/struct.RangeInclusive.html